### PR TITLE
feat: Redesign beta site

### PR DESCRIPTION
**Pull request by sarahmonster**
I think this is finally ready for review! It's ... a lot better than what's currently up there.

beta-site copy 7

### CODE REVIEW

File: data/theme.js [hunk 1]
```diff
@@ -23,16 +23,16 @@ const colors = {
...
+  purple300: '#BE87CC',
   purple400: '#8a71a2',
   purple500: '#4c2570',
-  purple600: '#432063',
+  purple600: '#6A008C',
   purple700: '#3a1c55',
-  purple800: '#2e1644',
-  purple900: '#1e0e2c',
+  purple800: '#3D0059',
+  purple900: '#1E0030',
```
Comment (tofumatt) [inline data/theme.js:35]: _UPPERCASE_? 😆

File: src/components/ButtonStyled/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { Button, useTheme } from '@octopusthink/nautilus';
+import React from 'react';
+import { css } from '@emotion/core';
+
+const ButtonStyled = (props) => {
```
Comment (tofumatt) [inline src/components/ButtonStyled/index.js]: "ButtonStyled" is a bit of an odd component name, maybe `TurnipButton` or something? I dunno, even just `Button` seems better 🤷🏻‍♂️

File: src/components/SignupForm/index.js [hunk 3]
```diff
@@ -1,57 +1,77 @@
...
-    >
-      <input type="hidden" value="1" name="embed"></input>
-      <input type="hidden" name="tag" value="turnip-beta"></input>
-      <div
+    <React.Fragment>
+      <form
+        action="https://buttondown.email/api/emails/embed-subscribe/octolabs"
+        method="post"
+        target="popupwindow"
+        onSubmit="window.open('https://buttondown.email/octolabs', 'popupwindow')"
```
Comment (tofumatt) [inline src/components/SignupForm/index.js:14]: 😱

I'll tweak this before we ship this 😅

File: src/components/SiteFooter/index.js [hunk 4]
```diff
@@ -1,21 +1,56 @@
...
       `}
     >
-      <Paragraph light noMargin small>
+      <Paragraph
+        light
+        noMargin
+        small
+        css={css`
+          ${heading.small(theme)};
+          color: #edf0ee;
```
Comment (tofumatt) [inline src/components/SiteFooter/index.js:43]: Should this be a variable or something instead?

File: src/components/SiteFooter/index.js [hunk 5]
```diff
@@ -1,21 +1,56 @@
...
+          margin: 2.4rem auto;
+          max-width: ${config.siteContentWidth};
+          position: relative;
+
+          a {
+            box-shadow: none;
+            color: #edf0ee;
```
Comment (tofumatt) [inline src/components/SiteFooter/index.js:50]: Variable as well?

**Review by tofumatt — COMMENTED** [review]
Largely looks good, but I want to tweak the buttondown `onSubmit` before we ship 👍🏻

**Comment by tofumatt** [comment]
Oh, one note: the hover colour on the "Octopus Think" link on the footer is a bit dark/low contrast.

File: src/components/SiteFooter/index.js (hunk 4 above)
Comment (sarahmonster) [reply src/components/SiteFooter/index.js]: Sure should! I've fixed it.

File: data/theme.js (hunk 1 above)
Comment (sarahmonster) [reply data/theme.js]: That's how they come out of Sketch! But I manually fixed them.

File: src/components/ButtonStyled/index.js (hunk 2 above)
Comment (sarahmonster) [reply src/components/ButtonStyled/index.js]: You fixed this already, but noted on the approach—makes more sense! 👍

File: src/components/SignupForm/index.js (hunk 3 above)
Comment (sarahmonster) [reply src/components/SignupForm/index.js:14]: Technically I'm not sure it does anything—we might be able to remove it. 

For reference, how we're doing it elsewhere...

https://github.com/octopusthink/octopusthink.com/blob/master/src/components/NewsletterSignupForm/index.js
https://github.com/octopusthink/getmicdrop.com/blob/master/src/components/NewsletterSignupForm/index.js

...should I just copy the approach used there?

**Comment by sarahmonster** [comment]
All fixed!

**Review by tofumatt — APPROVED** [review]
😍

Let's ship it 👍🏻

**Comment by tofumatt** [comment]
We'll want to update the icons on octopusthink.com as well, given they currently look like this:

Screenshot 2021-02-06 at 12 21 08

Screenshot 2021-02-06 at 12 21 27

But that can be separate. For now, let's ship this.

**Pull request merged**